package.json

```
@@ -55,80 +55,61 @@
+    "@storybook/addon-interactions": "^8.3.0",
+    "@storybook/addon-links": "^8.3.0",
+    "@storybook/react": "^8.3.0",
+    "@storybook/react-vite": "^8.3.0",
+    "@swc/core": "^1.7.26",
+    "@testing-library/jest-dom": "^6.5.0",
+    "@testing-library/react": "^16.0.1",
     "@testing-library/user-event": "^14.5.2",
-    "@types/node": "18.17.12",
+    "@types/node": "22.5.5",
     "@types/react": "18.2.21",
     "@types/react-dom": "18.2.7",
-    "@typescript-eslint/eslint-plugin": "5.62.0",
-    "@typescript-eslint/parser": "5.62.0",
+    "@typescript-eslint/eslint-plugin": "7.18.0",
+    "@typescript-eslint/parser": "7.18.0",
     "@ubie/prettier-config": "0.1.0",
-    "@vitejs/plugin-react": "^4.2.1",
-    "@vitejs/plugin-react-swc": "^3.6.0",
-    "autoprefixer": "10.4.15",
-    "babel-loader": "9.1.3",
-    "chalk": "5.3.0",
-    "clsx": "2.0.0",
-    "css-loader": "6.8.1",
-    "eslint": "8.48.0",
-    "eslint-config-prettier": "9.0.0",
-    "eslint-import-resolver-typescript": "3.6.0",
-    "eslint-plugin-import": "2.28.1",
-    "eslint-plugin-jsx-a11y": "6.7.1",
-    "eslint-plugin-react": "7.33.2",
+    "@vitejs/plugin-react": "^4.3.1",
+    "@vitejs/plugin-react-swc": "^3.7.0",
+    "clsx": "2.1.1",
+    "eslint": "8.56.0",
+    "eslint-config-prettier": "9.1.0",
+    "eslint-import-resolver-typescript": "3.6.3",
+    "eslint-plugin-import": "2.30.0",
+    "eslint-plugin-jsx-a11y": "6.10.0",
+    "eslint-plugin-react": "7.36.1",
     "eslint-plugin-storybook": "^0.8.0",
-    "eslint-plugin-unused-imports": "2.0.0",
-    "execa": "8.0.1",
-    "glob": "10.3.3",
-    "husky": "8.0.3",
-    "jsdom": "^24.0.0",
-    "lint-staged": "14.0.1",
-    "markuplint": "^4.4.0",
+    "eslint-plugin-unused-imports": "3.2.0",
+    "jsdom": "^25.0.0",
+    "markuplint": "^4.9.3",
     "npm-run-all": "4.1.5",
-    "playwright": "^1.38.1",
-    "prettier": "3.0.2",
+    "playwright": "^1.47.1",
+    "prettier": "3.3.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "release-it": "^17.1.1",
-    "rollup": "3.28.1",
-    "rollup-plugin-filesize": "^10.0.0",
-    "rollup-plugin-postcss": "4.0.2",
-    "scaffdog": "3.0.0",
-    "storybook": "^8.1.1",
-    "style-loader": "3.3.3",
-    "stylelint": "^16.2.0",
-    "stylelint-config-recess-order": "^5.0.1",
-    "stylelint-config-standard": "^36.0.0",
-    "tsup": "^7.2.0",
-    "typescript": "5.2.2",
-    "vite": "^4.4.9",
-    "vitest": "^0.34.3"
+    "release-it": "^17.6.0",
+    "scaffdog": "4.0.0",
+    "storybook": "^8.3.0",
+    "stylelint": "^16.9.0",
+    "stylelint-config-recess-order": "^5.1.0",
+    "stylelint-config-standard": "^36.0.1",
+    "tsup": "^8.2.4",
+    "typescript": "5.6.2",
+    "vite": "^5.4.5",
+    "vitest": "^2.1.1"
   },
   "peerDependencies": {
     "@headlessui/react": ">1.7.0 <2.0.0",
     "@ubie/design-tokens": ">=0.2.0",
     "@ubie/ubie-icons": ">=0.5.0 <0.6.2 || >=0.8.0",
-    "clsx": ">1.2.0",
     "react": "^17 || ^18",
     "react-dom": "^17 || ^18"
```

```suggestion
    "react-dom": "^18"
```

https://github.com/ubie-oss/ubie-ui/pull/135/files#r1760927832 thks @kodai3 

It now requires 18 or more.